bin/workloads/sql/common/set_hive_env.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+if [[ $HADOOP_HOME =~ "3.2" || $HADOOP_HOME =~ "3.1" ]];then
```

Comment:
Could we maybe add some comments here about why the script is necessary? This can help our future maintenance.
For example, why for Hadoop 3.0 + we need to additionally call `chematool -initSchema -dbType derby`?

Reply:
@gczsjdy  That's good idea.